kie-internal/src/main/java/org/kie/internal/xstream/LocalDateXStreamConverter.java

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright 2017 Red Hat, Inc. and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+*/
+
+package org.kie.internal.xstream;
+
+import java.time.LocalDate;
+
+import com.thoughtworks.xstream.converters.Converter;
+import com.thoughtworks.xstream.converters.MarshallingContext;
+import com.thoughtworks.xstream.converters.UnmarshallingContext;
+import com.thoughtworks.xstream.io.HierarchicalStreamReader;
+import com.thoughtworks.xstream.io.HierarchicalStreamWriter;
+
+public class LocalDateXStreamConverter implements Converter {
```

Comment:
This belongs in XStream, let's see what this discussion teaches us:
  https://groups.google.com/forum/#!topic/xstream-user/IXAkhHVdOGU

I suspect it doesn't support it yet (surprising to me!), so we'll need to accept this Converter here temporarily, but create an issue on XStream, add a comment here like "temporarly workaroudn until ...xstreamIssueUrl... is fixed" and as soon as XStream support it, we upgrade the xstream dependency and remove this class.